frontend/src/metabase/services.js

```diff
@@ -28,6 +32,35 @@ export const GTAPApi = {
   attributes: GET("/api/mt/user/attributes"),
 };
 
+export function maybeUsePivotEndpoint(api: APIMethod, card: Card): APIMethod {
```

Comment:
Were there any other approaches you considered to this? It feels a bit messy to have to go through this code path for cards that aren't displayed as pivots.